@here/harp-mapview/lib/MapView.ts

```diff
@@ -1738,6 +1742,10 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_env;
     }
 
+    setDynamicProperty(name: string, value: Value) {
+        this.m_env.entries[name] = value;
+    }
+
```

Comment:
I would prefer to keep the keep the MapView environment read-only and maybe add APIs to DataSource to set/update envs.